createdatabase.sql

```diff
@@ -1,18 +1,22 @@
+CREATE DATABASE  IF NOT EXISTS community-org-finder;
+USE community-org-finder;
+
+DROP TABLE IF EXISTS community_orgs;
 CREATE TABLE community_orgs (
     id INT PRIMARY KEY AUTO_INCREMENT,
     community_org_name VARCHAR(40) UNIQUE NOT NULL,
     telephone VARCHAR(15),
     email VARCHAR(50) UNIQUE,
-    address VARCHAR(70),
-    longitude DECIMAL,
-    latitude DECIMAL
```

Comment:
why have you removed these lines?

Reply:
I have no idea???  I don't remember doing that so I presume it was by mistake!